mantidproject/mantid
# rounding in muon GUI

**Description of work.**

This changes the rounding in the muon GUI to be 6dp for the alpha values of a pair. This includes when using the "guess alpha" button. 

I have also updated the run info to be to 3 dp for numerical logs. The strangely high precision was confusing users.

**To test:**


Open muon analysis
Set the instrument to MUSR
Load 76811
At the bottom of the home tab, look at the run info and check that the sample temp is to 3 dp
Go to the grouping tab
Press calculate alpha
Check that the result is to 6 dp
Try to type in a value for alpha that has more than 6 dp
Check that the typed value gets saved to 6 dp

Fixes #32799 and #32720. 



Added to release notes


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
